Services/Init/templates/default/tpl.startup_screen.html

```diff
@@ -16,6 +16,15 @@ <h5 class="ilAccHeadingHidden"><a id="il_message_focus" name="il_message_focus">
 	</div>
 	<!-- END success -->
 </div>
+
```

Comment:
@nhaagen das müsste vermutlich anders gelöst werden, ich würde dies als Topbar-item in der neuen `StartUpModificationProvider` machen

Reply:
Eine LanguageSelection-Slate?